_maps/map_files/Mining/Lavaland.dmm

```diff
@@ -3708,6 +3708,7 @@
 /area/mine/production)
 "rP" = (
 /obj/machinery/atmospherics/pipe/layer_manifold{
+	icon_state = "manifoldlayer";
```

Comment:
unneeded change,please remove
```suggestion

```

Reply:
@alexkar598 Dunno how the fuck that happened since i didnt touched normal mining but ok